crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -440,12 +437,23 @@ impl BeaconState {
             exit_queue_epoch += 1;
         }
 
-        // Set validator exit epoch and withdrawable epoch
-        if let Some(validator) = self.validators.get_mut(index as usize) {
-            validator.exit_epoch = exit_queue_epoch;
-            validator.withdrawable_epoch =
-                validator.exit_epoch + MIN_VALIDATOR_WITHDRAWABILITY_DELAY;
+        let validator = self
+            .validators
+            .get_mut(index as usize)
+            .expect("could not get validator");
```

Comment:
```suggestion
            .expect("could not get validator");
```

We shouldn't use expect and instead use an anyhow error